force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -237,13 +270,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return A Set of unique strings.
      */
     public static Set<String> createStringSet(List<String> inputList) {
-        // Create a new Set of strings
+        // createStringSet is method name to create new empty set called stringSet
         Set<String> stringSet = new Set<String>();
+            // Go thru each word one by one from provided inputList
+            for (String myString : inputList) {
+                // Keep each word on the list only once
+                stringSet.add(myString);
+            }
+            return stringSet;  //Passes Debug
+        }
```

Comment:
another way is to pass the list into the Set constructor directly. You can solve this problem with one line of code.